projects/plugins/jetpack/modules/masterbar/masterbar/class-masterbar.php

```diff
@@ -127,6 +132,7 @@ public function __construct() {
 		$this->is_rtl          = isset( $this->user_data['text_direction'] ) && 'rtl' === $this->user_data['text_direction'];
 		$this->user_locale     = isset( $this->user_data['user_locale'] ) ? $this->user_data['user_locale'] : '';
 		$this->site_woa        = ( new Host() )->is_woa_site();
+		$this->use_dev_nav     = $this->site_woa && ( isset( $this->user_data['wpcom_global_nav_enabled'] ) ? $this->user_data['wpcom_global_nav_enabled'] : false );
```

Comment:
hmm is `dev_nav` the name we want to go with? I think we need to clarify the name of this navigation bar style. It could be "classic nav" potentially or "global nav", what do you think?

Reply:
i didn't give it much thought 😅. Im still unsure (as it seems others are as well) at this point if this feature is going to be for people selecting the 'developer friendly features' toggle, the classic view toggle, or be a global nav for everyone. 

But yes, we can find something more fitting for this and the corresponding css.

Reply:
Hmmm it looks like people are gravitating towards calling it "Classic Nav" but I am personally concerned that calling it "classic" might cause confusion with the "Classic View" hosting-configuration option. pe7F0s-1I7-p2#comment-2218